be/src/formats/parquet/column_chunk_reader.h

```diff
@@ -121,10 +113,9 @@ class ColumnChunkReader {
     level_t _max_rep_level = 0;
     int32_t _type_length = 0;
     const tparquet::ColumnChunk* _chunk_metadata = nullptr;
-    ColumnChunkReaderOptions _opts;
-    RandomAccessFile _file;
-
+    const ColumnReaderOptions& _opts;
```

Comment:
Are you sure that the option's life is longer than the Reader?

Reply:
It must be longer than `ColumnChunkReader`.  The `ColumnReaderOptions` is hold by `GroupReader`.